docs/api/users.md

```diff
@@ -51,3 +51,47 @@ Content-Type: application/json
   "updated": "2016-12-22T02:14:05Z"
 }
 ```
+
+
+## List users
+
+### URL
+
+```http
+GET /users HTTP/1.1
+```
```

Comment:
Does this call list all users? Only users of the current Organisation from the OAuth scope? Or Organisation from a given parameter?